docs/AVM_Specification.md

```diff
@@ -265,9 +265,9 @@ The instructions are as follows:
 | 0xa1 | getbuffer8 | Pop two values (A) and (B) off the stack. The value A must be a buffer and B must be an integer smaller than 2**64. Pushes to the stack Bth byte of buffer A. | 10 |
 | 0xa2 | getbuffer64 | Pop two values (A) and (B) off the stack. The value A must be a buffer and B must be an integer smaller than 2**64-7. Pushes to the stack B..B+7 bytes of buffer A as BE integer. | 10 |
 | 0xa3 | getbuffer256 | Pop two values (A) and (B) off the stack. The value A must be a buffer and B must be an integer smaller than 2**64-31. Pushes to the stack B..B+31 bytes of buffer A as BE integer. | 10 |
-| 0xa4 | setbuffer8 | Pop three values (A), (B) and (C) off the stack. The value A must be a buffer, B must be an integer smaller than 2**64 and C must be an integer. Pushes to stack a new buffer that is same as A except that byte in position B is now C. | 100 |
-| 0xa5 | setbuffer64 | Pop three values (A), (B) and (C) off the stack. The value A must be a buffer, B must be an integer smaller than 2**64-7 and C must be an integer. Pushes to stack a new buffer that is same as A except that bytes in positions B..B+7 are now BE representation of C as 64-bit integer. | 100 |
-| 0xa6 | setbuffer256 | Pop three values (A), (B) and (C) off the stack. The value A must be a buffer, B must be an integer smaller than 2**64-31 and C must be an integer. Pushes to stack a new buffer that is same as A except that bytes in positions B..B+31 are now BE representation of C as 256-bit integer. | 100 |
+| 0xa4 | setbuffer8 | Pop three values (A), (B) and (C) off the stack. The value A must be a buffer, B must be an integer smaller than 2**64 and C must be an integer smaller than 2**8. Pushes to stack a new buffer that is same as A except that byte in position B is now C. | 100 |
```

Comment:
I think that this should explicitly state that otherwise this should error. Same for the other opcodes of course